src/test/java/uk/gov/hmcts/reform/blobrouter/reconciliation/service/ReconciliationMailServiceTest.java

```diff
@@ -416,7 +416,7 @@ void should_send_summary_report_mail_when_reconciliation_report_has_just_summary
         TargetStorageAccount account,
         ReconciliationReport reconciliationReport,
         SummaryReport summaryReport
-    ) throws IOException, SendEmailException {
+    ) throws Exception {
```

Comment:
why?

Reply:
A bit not related to the task but we usually declare generic exceptions in test methods. Can revert for now but later I think it should be changed.